libtelemetry/src/test/java/com/mapbox/android/telemetry/MapboxTelemetryTest.java

```diff
@@ -121,7 +121,30 @@ public void checksOnFullQueueSendEventsNotCalledWhenIsNotConnected() throws Exce
 
   @Test
   public void checksOnEventReceivedPushCalledWhenTelemetryEnabled() throws Exception {
-    Context mockedContext = mock(Context.class);
+    Context mockedContext = mock(Context.class, RETURNS_DEEP_STUBS);
+    MapboxTelemetry.applicationContext = mockedContext;
+    String aValidAccessToken = "validAccessToken";
+    String aValidUserAgent = "MapboxTelemetryAndroid/";
+    EventsQueue mockedEventsQueue = mock(EventsQueue.class);
+    TelemetryClient mockedTelemetryClient = mock(TelemetryClient.class);
+    Callback mockedHttpCallback = mock(Callback.class);
+    SchedulerFlusher mockedSchedulerFlusher = mock(SchedulerFlusher.class);
+    Clock mockedClock = mock(Clock.class);
+    LocalBroadcastManager mockedLocalBroadcastManager = mock(LocalBroadcastManager.class);
+    MapboxTelemetry theMapboxTelemetry = new MapboxTelemetry(mockedContext, aValidAccessToken, aValidUserAgent,
+      mockedEventsQueue, mockedTelemetryClient, mockedHttpCallback, mockedSchedulerFlusher, mockedClock,
+      mockedLocalBroadcastManager);
+    Event mockedEvent = mock(Event.class);
+    theMapboxTelemetry.enable();
+
+    theMapboxTelemetry.onEventReceived(mockedEvent);
+
+    verify(mockedEventsQueue, times(1)).push(eq(mockedEvent));
+  }
+
+  @Test
+  public void checksOnEventReceivedPushCalled() throws Exception {
```

Comment:
This test is doing the same exact thing as `checksOnEventReceivedPushCalledWhenTelemetryEnabled` and should be removed. It doesn't add any value.